### DISCUSSION TITLE
User preview

### DISCUSSION THREAD

**Discussion by Reebooted21** [ScratchAddons/ScratchAddons#1391] [Ideas]
Can you add a user preview so that if you hover your mouse over someone's username, it shows a preview of their profile page like this ? :

![image](https://user-images.githubusercontent.com/70284560/105607392-6e880500-5daf-11eb-8923-d080e7db186a.png)

**Comment by gazalied** [comment]
What would it show?

**Reply by mxmou** [reply]
Probably the profile picture and About me.

**Reply by gazalied** [reply]
let's say the about me is long.

**Reply by mxmou** [reply]
> let's say the about me is long.

It can't be longer than 250 characters, which is quite short.

**Comment by gazalied** [comment]
any mockups from "mockup artists"?

**Reply by retronbv** [reply]
I could make one

**Reply by retronbv** [reply]
![37E26A59-5DD6-447D-BC89-3585003B0010](https://user-images.githubusercontent.com/49005044/107034318-8afb4880-6784-11eb-8132-46090ab67cd5.jpeg)
This took me around 10 mins
